src/client/testing/page-objects/create-availability.ts

```diff
@@ -3,11 +3,103 @@ import RootPage from './root';
 
 export default class CreateAvailabilityPage extends RootPage {
   readonly title: Locator;
+  readonly btnCreateAvailability: Locator;
+  readonly sessionTittle: Locator;
+  readonly btnContinue: Locator;
+  readonly setAndCapacityTittle: Locator;
+  readonly btnSaveSession: Locator;
+  readonly sessionSuccessMsg: Locator;
 
   constructor(page: Page) {
     super(page);
     this.title = page.getByRole('heading', {
       name: 'Create availability',
     });
+
+    this.btnCreateAvailability = page.getByRole('button', {
+      name: 'Create availability',
+    });
+
+    this.sessionTittle = page.getByRole('heading', {
+      name: 'What type of session do you want to create?',
+    });
+
+    this.btnContinue = page.getByRole('button', {
```

Comment:
This is a very minor thing and just my opinion, so feel free to ignore me if you want, but in the other page object models we're trying to describe components the way you would in a real life sentence, for example `logOutButton`. I'd personally rather stick to this pattern rather than start adding `btn` to the start. 

Personally I find `continueButton.click()` easier to read than `btnContinue.click()`, but like I say that's just my opinion! 

It'd also be nice to keep things consistent, so if we do decide to go down this route then we should rename the other models in kind (i.e. change `logOutButton` to `btnLogOut` )